### PR TITLE
Documentation update - Added theme switching param to websdk

### PR DESCRIPTION
**Pull request by akbar-idwise**

### CODE REVIEW

File: web-sdk.md [hunk 1]
```diff
@@ -65,6 +67,7 @@ Please follow these steps to integrate with the document capturing SDK:
                   idwise = await IDWise.initialize({
                     clientKey: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=',
                     locale: 'en',
+                    theme: 'dark',
```
Comment (mheskol) [inline web-sdk.md:70]: Can we add a comment here that if this is left out it will follow the system?

**Review by mheskol — APPROVED** [review]

**Review by sakher — APPROVED** [review]
Left just a small comment, otherwise looks great

File: web-sdk.md (hunk 1 above)
Comment (mheskol) [reply web-sdk.md:70]: What is status of this PR @akbar-idwise ?

Comment (akbar-idwise) [reply web-sdk.md]: Its updated, can you please approve so we can merge?

**Pull request merged**